src/utils/dom.js

```diff
@@ -112,6 +112,6 @@ export function scrollIntoView(itemEl: ?HTMLElement): void {
     // @NOTE: breaks encapsulation but alternative is unknown child ref
     if (itemEl) {
         const parentEl = itemEl.closest(`.body, .modal, .${OVERLAY_WRAPPER_CLASS}`);
-        scrollIntoViewIfNeeded(itemEl, false, undefined, parentEl);
+        scrollIntoViewIfNeeded(itemEl, { boundary: parentEl });
```

Comment:
The place that's using this 'selectorDropdown' does not seems to be doing anything even with the old code. Anyone know how the selectorDropdown suppose to behave originally?